blueprints/app/files/README.md

```diff
@@ -31,8 +31,8 @@ Make use of the many generators for code, try `ember help generate` for more det
 
 ### Running Tests
 
-* `ember test`
-* `ember test --server`
+* `<% if (yarn) { %>yarn test<% } else { %>npm test<% } %>`
```

Comment:
We already have linting mentioned below, so it's a bit redundant to mention the command that runs linting as well. On the other hand, how are people going to learn that `ember test` no longer lints, and that you should be running `npm test` to get the old behavior?

Also, we now have a weird mix of npm-scripts wrapped commands and native ember-cli commands. We may need learning team help on resolving this confusion. cc @jenweber @NullVoxPopuli @kennethlarsen

Reply:
> You mean yarn lint:js && ember test?

idk, I prefix everything with yarn, so I _know_ I'm using the local version

> It is already split in travis (or should be).

 #biwinning

Reply:
Perhaps the test output could indicate that linting is not included. For example, it could say afterwards "to run linting, use `npm test`." I will have to think about this some. 

Beginners aside, this is a tough question because I imagine that many people just do ember test in their CI. I don't use Travis and I'm pretty sure that I just run `ember test` in CI.